src/radical/pilot/agent/agent_0.py

```diff
@@ -503,7 +480,7 @@ def _start_sub_agents(self):
                         'ranks'         : 1,
                         'cores_per_rank': self._rm.info.cores_per_node,
                         'executable'    : '/bin/sh',
-                        'arguments'     : [bs_name, sa]
+                        'arguments'     : [bs_name, self._sid, self.cfg.reg_addr, sa]
```

Comment:
there is one more `bootstrap_2.sh` above (L455), where need to update arguments

Reply:
Thanks!